source/auth/tests/legacy/connection-string.yml

```diff
@@ -434,17 +434,6 @@ tests:
     mechanism_properties:
       ENVIRONMENT: azure
       TOKEN_RESOURCE: 'mongodb://test-cluster'
-- description: should handle a complicated url-encoded TOKEN_RESOURCE (MONGODB-OIDC)
-  uri: mongodb://user@localhost/?authMechanism=MONGODB-OIDC&authMechanismProperties=ENVIRONMENT:azure,TOKEN_RESOURCE:abc%2Cd%25ef%3Ag%26hi
-  valid: true
-  credential:
-    username: user
-    password: null
-    source: $external
-    mechanism: MONGODB-OIDC
-    mechanism_properties:
-      ENVIRONMENT: azure
-      TOKEN_RESOURCE: 'abc,d%ef:g&hi'
```

Comment:
Do we want to keep this test? Making sure that percent-encoded characters are properly decoded and that `:` is kept in the value seems valuable in general

Reply:
So just remove the comma then?

Reply:
Yeah, I think so?